python/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ include(rapids-cython-core)
 if(NOT ucxx_FOUND)
   set(BUILD_TESTS OFF)
   set(BUILD_BENCHMARKS OFF)
+  set(UCXX_ENABLE_PYTHON ON)
```

Comment:
I think this won't have any effect here as this flag is part of the `cpp/CMakeLists.txt` to build CPython code from `cpp/python/`.

Reply:
This flag will have an effect for wheel builds, which trigger a build of the C++ from the Python build.

Reply:
> This flag will have an effect for wheel builds, which trigger a build of the C++ from the Python build

Ok, this makes sense.

> or more generally in the case when the Python package build cannot find a pre-built libucxx

I'm not sure about what this means though, what if a pre-built libucxx isn't found? Is that essentially then saying "build libucxx too"?

Reply:
Yes, that's right. That's how all our packages are set up. That way in theory someone could take the source and do a pip install and it would just work. They don't have to know to build the cpp lib first. Of course, in most cases we work with we want to build the cpp lib first and find it.

Reply:
Oh I see, that's pretty clever. Thanks for the details!